PowerShellGet/PSModule.psm1

```diff
@@ -1411,8 +1496,10 @@ function Find-Module
                 $isRepositoryNullOrPSGallerySpecified = $true
             }
         }
-		
-		PackageManagement\Find-Package @PSBoundParameters | Microsoft.PowerShell.Core\ForEach-Object {
+        
+        $results = PackageManagement\Find-Package @PSBoundParameters

```

Comment:
Oh, this was used during debugging, I needed to see what the results were.  It is not necessary, I will put it back to how it was before.

---
In reply to: [132600289](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132600289) [](ancestors = 132600289)